### PR TITLE
Close- initial html and css files

### PR DESCRIPTION
**Pull request by sruble**
Hi everyone, here is the initial html and css. There are still a few glitches that need to be worked out with css.

### CODE REVIEW

**Comment by sruble** [comment]
the css file isn't connected, I'm currently trying to see where I went wrong with this.

**Review by weberwilliamb — COMMENTED** [review]
Is the CSS connecting to the HTML correctly?

**Comment by RoEric** [comment]
I think your html file should be a cshtml file to support Razor.

**Comment by sruble** [comment]
> I think your html file should be a cshtml file to support Razor.

thanks for caching that, Eric.

**Review by MaxIgnatyev — APPROVED** [review]
Looks good.

File: photo-points/Views/Home/WelcomeAdmin.html [hunk 1]
```diff
@@ -48,19 +50,23 @@ <h1>Welcome, &#91;Administrator&#93;</h1>
...
-                    <div>
-                        <li><a href="PendingPhotos.html">Pending Photos</a></li>
-                    </div>                     
+                     <div>
+                         <!--<li><a href="PendingPhotos.html">Pending Photos</a></li>-->
+                         <button type="button" onclick="location.href = '/PendingPhotos.html';" value="Pending Photos">Pending Photos</button>
+                     </div>                     
                 </article>
                 
                 <article class="menu-one">
```
Comment (weberwilliamb) [inline photo-points/Views/Home/WelcomeAdmin.html:65]: Is this supposed to have the same article class as line 51?

**Pull request merged**